app/models/user.rb

```diff
@@ -14,6 +14,8 @@ class User < ActiveRecord::Base
     :two_factor_authenticatable,
     :omniauthable,
     authentication_keys: [:email],
+    case_insensitive_keys: [],
+    strip_whitespace_keys: [],
```

Comment:
What do you think about making these changes in the Devise config instead?https://github.com/18F/identity-idp/blob/master/config/initializers/devise.rb#L7